openmc/deplete/abc.py

```diff
@@ -1004,12 +1012,23 @@ def _get_bos_data_from_operator(self, step_index, step_power, bos_conc):
             self.operator.settings.particles //= self.n_steps
         return inherited
 
-    def integrate(self):
-        """Perform the entire depletion process across all steps"""
+    def integrate(self, output=True):
+        """Perform the entire depletion process across all steps
+
+        Parameters
+        ----------
+        output : bool, optional
+            Indicate whether to display information about progress
+
+            .. versionadded:: 0.13.1
+        """
         with self.operator as conc:
             t, self._i_res = self._get_start_data()
 
             for i, (dt, p) in enumerate(self):
+                if output:
+                    print(f"[openmc.deplete] t={t} s, dt={dt} s, source={p}")
+
```

Comment:
This will check if output is True with every iteration of the loop, perhaps there are more efficient methods of doing the verbose, but I think this is fine for now

Reply:
Each iteration of this loop usually involves running an OpenMC transport solve at least once, so the cost of a single if/print is negligible here